packages/paste-utils/src/hooks/useMutationObservable.ts

```diff
@@ -1,7 +1,7 @@
 import {useEffect, useState} from 'react';
 
 // This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export
```

Comment:
Question: is this comment still relevant?

Reply:
It is not!